raiden-dapp/src/views/Transfer.vue

```diff
@@ -63,7 +63,7 @@
       @submit.prevent="navigateToTransferSteps(target, amount)"
     >
       <v-row justify="center" align="center" class="transfer__recipient">
-        <v-col cols="12" sm="10">
```

Comment:
I think it worked before with `cols="12" sm="10"` as well, but I accidentally moved the `v-form` next to a row which seems to cause this issue. Looks like vuetify only wants v-rows to have v-row siblings

Reply:
Yes, it could possibly be it. I didn't spend more time looking into it as the change seems to have the required result :)

Reply:
Yes no big deal =)